style.css

```diff
@@ -0,0 +1,64 @@
+
+body {
+  font-family: 'Popins', sans-serif;
+  margin: 0;
+  min-height: 100vh;
+}
+
+header {
+  height: 10%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.mylogo {
+  width: 76px;
+  height: 20px;
+  margin-left: 8px;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+}
+
+.headerline {
+  background: url(./assets/img/header-shapes\ mobile.png);
+  height: 90%;
+  display: flex;
+  flex-direction: column;
+}
+
+.headerline-detail {
+  margin: 80px 12px 0 12px;
+  width: 327px;
+}
+
+.name {
+  height: 150px;
+  font-weight: 800;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+.aboutme {
+  margin: 0;
+  font-weight: 400;
+  color: #344563;
+}
+
+.connect {
+  font-weight: 500;
+  color: #7f8cff;
+  text-transform: uppercase;
+}
+
+.social-media-icons {
+  list-style-type: none;
+  padding-left: 0;
+  width: 170px;
+  height: 20px;
+  display: flex;
+  justify-content: space-between;
+}
```

Comment:
- [ ] To match the Figma design your social icons should be aligned center, a quick fix to this is to add
```CSS
align-items:center
```
to your  `.social-media-icons` class